internal/command/project_application_key.go

```diff
@@ -61,7 +88,10 @@ func (c *Commands) AddApplicationKey(ctx context.Context, key *domain.Applicatio
 	if err != nil {
 		return nil, err
 	}
-	result := applicationKeyWriteModelToKey(keyWriteModel, key.PrivateKey)
+	result := applicationKeyWriteModelToKey(keyWriteModel)
+	if key.PrivateKey != nil && len(key.PrivateKey) > 0 {
```

Comment:
```suggestion
	result := applicationKeyWriteModelToKey(keyWriteModel)
	if len(key.PrivateKey) > 0 {
```

Reply:
or you could still just pass it to the applicationKeyWriteModelToKey function as it's just nil